src/common/components/elements/ProfileMenu.tsx

```diff
@@ -35,6 +43,13 @@ export const ProfileMenu: FC = () => {
               >
                 <FeatherIcon height={16} width={16} icon="image" className="mr-4" />
                 Owned
+                {selected && (
+                  <span
+                    className={cx(`ml-1`, selected ? `font-bold text-rose-400` : `text-gray-300`)}
```

Comment:
I don't think we use rose anymore, just black, white and gray. I'm sure Belle will have this specified in the final designs :)